orion/sites/admin/src/achieve/achieveEdit.styl

```diff
@@ -0,0 +1,50 @@
+@import "../static.styl"
+
+#view-achieve-edit
+  width: spacing96x
+  padding: spacing1x spacing8x spacing4x
+  box-sizing: border-box
+  display: 
```

Comment:
missing value here

Reply:
and if you want a `display` value, it should be defined above `width` and `padding`.